src/main/kotlin/com/exactpro/th2/codec/DecodeProcessor.kt

```diff
@@ -70,6 +70,11 @@ class DecodeProcessor(
                 }
 
                 messageBatch.addGroups(decodedGroup)
+            } catch (e: ValidateException) {
+                parentEventId.onEachErrorEvent(
+                    "Failed to decode message group due to a message: ${e.title}", messageGroup.messageIds, e
```

Comment:
Let's make the constant name part a little bit shorter
```suggestion
                    "Failed to decode: ${e.title}", messageGroup.messageIds, e
```